### ISSUE TITLE
High Hz Monitors

### ISSUE THREAD

**Issue by manic232** [wipeout-phantom-edition/wipeout-phantom-edition#30]
I have a 360hz monitor and I have noticed that on certain tracks that have moving objects like Arridos IV for example (mining equipment I think?) the objects move a silly speeds, not sure if this is an easy fix of not, also not sure if this is related but the latest update does not seem to resect the vsync setting, even with vsync on the game runs a some alarming fps on my system (over 2000 fps) this make the moving object move a ludicrous speeds.

Just one more thing whilst I'm here, I know this has been touched on in a previous post, but I would LOVE an option to do away with draw distance, I am not sure if there is a technical reason why draw distance and the fading in of the scenery still needs to be there or not but if that could be removed then that really would be the icing on the cake!

Love this port, truly awe inspiring!

Michael.

**Comment by wipeout-phantom-edition** [comment]
You are right, those objects in Arridos IV have not had their movement properly decoupled from the render frames, I'll fix that. I can't reproduce any issue with vsync not working though.
